commands/utils/SourceArchiveUtility.ts

```diff
@@ -0,0 +1,114 @@
+import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry/lib/containerRegistryManagementClient';
+import { DockerBuildRequest, FileTaskRunRequest, Registry } from 'azure-arm-containerregistry/lib/models';
+import { Subscription } from 'azure-arm-resource/lib/subscription/models';
+import { BlobService, createBlobServiceWithSas } from "azure-storage";
+import * as fse from 'fs-extra';
+import * as os from 'os';
+import * as tar from 'tar';
+import * as url from 'url';
+import vscode = require('vscode');
+import { IActionContext, IAzureQuickPickItem } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { getBlobInfo, getResourceGroupName, streamLogs } from "../../utils/Azure/acrTools";
+import { AzureUtilityManager } from '../../utils/azureUtilityManager';
+import { Item } from '../build-image';
+import { quickPickACRRegistry, quickPickSubscription } from './quick-pick-azure';
+import { quickPickDockerFileItem, quickPickImageName, quickPickYamlFileItem } from './quick-pick-image';
+import { quickPickWorkspaceFolder } from './quickPickWorkspaceFolder';
+
+const idPrecision = 6;
+const vcsIgnoreList = ['.git', '.gitignore', '.bzr', 'bzrignore', '.hg', '.hgignore', '.svn']
+const status1 = vscode.window.createOutputChannel('Scheduled ACR Run');
```

Comment:
Not sure we need a separate output channel.  How about just using the existing ext.outputChannel (titled as 'Docker')?

Reply:
I just confirmed that. Consider it Done.